package.json

```diff
@@ -55,6 +55,7 @@
     "webpack-dev-server": "^1.16.2"
   },
   "dependencies": {
+    "lodash": "^4.17.11",
```

Comment:
Another option would be to use `https://github.com/jonschlinkert/clone-deep` if lodash is a bit too big of a dependency.